src/SFA-DAS-HMRC-API-Stub/Commands/GetFractionsCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SFA.DAS.HMRC.API.Stub.Data.Repositories;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.HMRC.API.Stub.Commands
+{
+    public class GetFractionsCommand : ICommand<GetFractionsRequest, GetFractionsResponse>
+    {
+        private readonly IFractionsRepository _fractionsRepository;
+
+        public GetFractionsCommand(IFractionsRepository fractionsRepository)
+        {
+            _fractionsRepository = fractionsRepository ?? throw new ArgumentException("FractionsRepository cannot be null");

```

Comment:
FractionsRepository in the ArgumentException message should have a lower case "f" to match the parameter passed into the constructor